### DISCUSSION TITLE
正版 Xbox 购买链接的那些事

### DISCUSSION THREAD

**Discussion by MoYuan-CN** [Meloong-Git/PCL#2438] [讨论]
早上在群里解答群友问题的时候，群友扔了个截图给我
![1@X%I1M2L$01U92I%B6X~J0](https://github.com/Hex-Dragon/PCL2/assets/109946426/555df2e0-29af-4bc7-922e-13cd5accda7f)
是不是意味着Xbox也挂了

**Comment by WForst-Breeze** [comment]
应该没挂，把系统区域调成美国试试？🤔

**Comment by SqrtTan** [comment]
刚看，Xbox没挂，还能用

**Comment by chenshaobao11** [comment]
Nice! I just got a Minecraft gift code for FREE! :D https://freeminecraftgiftcode.net

**Reply by iPodtouch6557** [reply]
Don't post off-topics
